requirements-dev-lite.txt

```diff
@@ -1,6 +1,8 @@
 boto3==1.17.106 # This should match the version in constraints-dev.txt
 flask>=1.0.0 # for s3 test only (with moto)
 freezegun>=0.3.15
+ipywidgets>=7.5.1
+mistune>=0.8.4,<2.0.0
```

Comment:
Shouldn't these also be removed from requirements.txt

Reply:
`ipywidgets` not currently in `requirements.txt`; `mistune` I believe is staying in but unpinned